src/libraries/System.Security.Cryptography.Primitives/src/System/Security/Cryptography/CryptoConfigForwarder.cs

```diff
@@ -4,11 +4,13 @@
 using System.Diagnostics.CodeAnalysis;
 using System.Reflection;
 
-namespace System.Security.Cryptography
-{
-    [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2026:RequiresUnreferencedCode",
+[assembly: UnconditionalSuppressMessage("ReflectionAnalysis", "IL2026:RequiresUnreferencedCode",
         Target = "M:System.Security.Cryptography.CryptoConfigForwarder.#cctor",
+        Scope = "member",
```

Comment:
(nit) whitespace? Should this be indented one to the left? Don't restart CI for it though.